agent/fw_drupal8.c

```diff
@@ -140,22 +140,26 @@ static void nr_drupal8_add_method_callback_before_after_clean(
     nrspecialfn_t after_callback,
     nrspecialfn_t clean_callback) {
   zend_function* function = NULL;
+  char* methodLC = NULL;
 
   if (NULL == ce) {
     nrl_verbosedebug(NRL_FRAMEWORK, "Drupal 8: got NULL class entry in %s",
                      __func__);
     return;
   }
 
-  function = nr_php_find_class_method(ce, method);
+  methodLC = nr_string_to_lowercase(method);
+  function = nr_php_find_class_method(ce, methodLC);
```

Comment:
Can we free `methodLC` after its use here and remove the 2 frees below?

Reply:
See cf07d9ae69c122c463a810396fd0e666c916f878.

Reply:
Now I think about it is there a reason for a temporary variable at all (aside from code clarity)?

Reply:
This looks good.